kernel/entries.ml

```diff
@@ -49,7 +49,7 @@ type one_inductive_entry = {
   mind_entry_lc : constr list }
 
 type mutual_inductive_entry = {
-  mind_entry_record : (Id.t option) option; 
+  mind_entry_record : (Id.t array option) option;
   (** Some (Some id): primitive record with id the binder name of the record
```

Comment:
The comment is outdated. And... `(Id.t array option) option`, seriously? The time has come for a dedicated type, I'm afraid.

Reply:
I can change the type into another PR. It is more related to the upper layers, so it would make sense to do this in the vernac-facing PR.